### PR TITLE
Exit form

### LINKED ISSUE #200: Implement Exit Form backbone!
Insert the following 3 views into the exit form:  
1. Did you have a good experience? (Y/N) 
2. Did you get the services you were looking for? (Y/N)
3. Do you have any feedback or comments? Anything you'd like to see? (Open-ended)

1 and 2 are probably best implemented via a checkbox view, but take a look at other views! 
3 is probably best implemented through an edittext view. 

Note that you don't have to implement actually saving the user input into the app for now.

### LINKED ISSUE #212: Checkout Form: change questions
Change questions to the following:
1. Rate your experience from 1-5. (should be array adapter) 
2. Of these services, what are your top 2 priorities? (choose from checkboxes)
3. Other comments? (this one's the same)

### PR DESCRIPTION
**Pull request by lilshim**
Fixes #212 and fixes #200

### CODE REVIEW

**Comment by chenhoward** [comment]
You have to put fixes before #200

**Comment by chenhoward** [comment]
Hi

PHC++

File: PHC-Android/PHC/src/main/java/phc/android/CheckoutFormFragment.java [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+        mCodeInputSubmitButton = (Button) view.findViewById(R.id.button_submit);
+        mCodeInputSubmitButton.setOnClickListener(new SubmitListener(getActivity()));
+        mCodeInput = (EditText) view.findViewById(R.id.text_code);
+
+        mRadioGroup = (RadioGroup) view.findViewById(R.id.checkout_radiogroup);
+        mRadioGroup.setOnCheckedChangeListener(new RadioGroup.OnCheckedChangeListener()
+        {
+            public void onCheckedChanged(RadioGroup group, int checkedId){
+                //Resource ids are not constants -> Can't use switch statements
+                if (checkedId==R.id.checkout_radio_0){
```
Comment (chenhoward) [inline PHC-Android/PHC/src/main/java/phc/android/CheckoutFormFragment.java]: Maybe do cases

File: PHC-Android/PHC/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -152,16 +152,24 @@
...
-    <string name="exit_services_title"> Did you receive the services you were looking for? </string>
-    <string name="exit_comment_title">Comments</string>
-    <string name="exit_yes">Yes</string>
-    <string name="hello_blank_fragment">Hello blank fragment</string>
+    <string name="scan_confirmation_texts">The scan result was: </string>
+
+    <!-- Checkout Activity -->
+
+    <string name="checkout_qr_scanner">Scan QR Code and Submit</string>
+    <string name="checkout_radio_group"></string>
```
Comment (Byronium) [inline PHC-Android/PHC/src/main/res/values/strings.xml]: is this necessary?

File: PHC-Android/PHC/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -152,16 +152,24 @@
...
+    <string name="checkout_experience_title">Rate your experience</string>
+    <string name="checkout_services_title"> Of these services, what are your top 2 priorities?</string>
+    <string name="checkout_comment_title">Other comments? </string>
+    <string name="checkout_hello_blank_fragment">Hello blank fragment</string>
```
Comment (Byronium) [inline PHC-Android/PHC/src/main/res/values/strings.xml]: what is this for?

File: PHC-Android/PHC/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -152,16 +152,24 @@
...
     <string name="checkout_another">Checkout another client</string>
+    <string name="checkout_service_blank">Sample Service</string>
+    <string name="checkout_experience_0">0</string>
+    <string name="checkout_experience_1">1</string>
+    <string name="checkout_experience_2">2</string>
+    <string name="checkout_experience_3">3</string>
+    <string name="checkout_experience_4">4</string>
+    <string name="checkout_experience_5">5</string>
+    <string name="hello_blank_fragment">Hello blank fragment</string>
```
Comment (Byronium) [inline PHC-Android/PHC/src/main/res/values/strings.xml]: and this?

File: PHC-Android/PHC/src/main/java/phc/android/CheckoutFormFragment.java [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+    protected String mScanResult;
+    /** The textview that holds the comment. */
+    private EditText mComment;
+    /** The checkboxview holds their input for good/bad experience */
+    private int mExperience;
+    /** The checkboxview holds their input for whether they found their services */
+    private CheckBox mServices;
+    /** Field and submit button for manual code input. **/
+    protected EditText mCodeInput;
+    protected RadioGroup mRadioGroup;
```
Comment (Byronium) [inline PHC-Android/PHC/src/main/java/phc/android/CheckoutFormFragment.java]: could ya add quick 1 line docstrings/comments for each field you added? see above for examples. 
also, you should probably just make these fields private for now - i doubt we'll be creating a subclass out of CheckoutFormFragment that'll need to inherit these protected fields.

File: PHC-Android/PHC/src/main/res/values/strings.xml (hunk 3 above)
Comment (lilshim) [inline PHC-Android/PHC/src/main/res/values/strings.xml]: they're all used for the checkout page. is there a specific one that doesn't make much sense?

File: PHC-Android/PHC/src/main/res/values/strings.xml (hunk 4 above)
Comment (lilshim) [inline PHC-Android/PHC/src/main/res/values/strings.xml]: used for the radio buttons on the checkout page

Comment (Byronium) [inline PHC-Android/PHC/src/main/res/values/strings.xml]: ah no i meant specifically line 172, and the other lines saying "Hello Blank Fragment". are these actually used?

Comment (lilshim) [inline PHC-Android/PHC/src/main/res/values/strings.xml]: everything is used except for 173, which i think was used for some testing i had. i deleted 173

**Pull request merged**